CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -497,4 +512,26 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return cell;
 }
 
+#pragma mark - UIViewControllerPreviewingDelegate
+
+- (nullable UIViewController *)previewingContext:(id <UIViewControllerPreviewing>)previewingContext viewControllerForLocation:(CGPoint)location {
```

Comment:
For our implementation files, we don't include nullability specifiers to encourage code legibility. 
It is just a coding guideline for RK and CK. Not anything particular to the implementation of the code itself.